pkg/metricfamily/override.go

```diff
@@ -0,0 +1,48 @@
+package metricfamily
+
+import (
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	client "github.com/prometheus/client_model/go"
+)
+
+// driftRange is used to observe timestamps being older than 5min, newer than 5min,
+// or within the present (+-5min)
+const driftRange = 5 * time.Minute
+
+var (
+	overwrittenMetrics = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "telemeter_overwritten_timestamps_total",
+		Help: "Number of timestamps that were in the past, present or future",
+	}, []string{"tense"})
+)
+
+func init() {
+	prometheus.MustRegister(overwrittenMetrics)
+}
+
+func OverwriteTimestamps(now func() time.Time) TransformerFunc {
```

Comment:
maybe let's add a little godoc for the future on why and what this transformer is doing.